__tests__/app.test.js

```diff
@@ -70,4 +70,43 @@ describe('2. GET /api/articles', () => {
         });
       });
   });
+});
+
+describe('3. GET /api/articles/:article_id', () => {
+  test('status:200, responds with an article object, with title, topic, author, body, created_at, and votes properties ', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body }) => {
+        const { article} = body;
+        expect(article).toBeInstanceOf(Object);
+          expect(article).toEqual(
+            expect.objectContaining({
+                article_id: 1,
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number)
```

Comment:
You could go a little further here to assert that all the properties are specific values as you know exactly what they should be from the test data. However, the article_id is the main one!